setup_testing_node.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -e
+
+if ! [ $(id -u) = 0 ] ; then
+	echo "$0 must be run as sudo user or root"
+	exit 1
+fi
+
+#We need to work around these default ip ranges, to make sure the tests pass
+#192.168.7.x = default usb0
+#192.168.8.x = default SoftAP0
+#192.168.9.x = (bbxm, work around 192.168.7.x usb0)
```

Comment:
I think I just realized that you added 192.168.10.x to specifically work-around the fix we already applied where xM will automatically move its client-provided address. Makes sense now.